python/py-immutables/Portfile

```diff
@@ -4,29 +4,31 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-immutables
-version             0.9
+version             0.20
 categories-append   devel
 license             Apache-2
 
-python.versions     35 36 37
+python.versions     38 39 310 311 312
```

Comment:
```
❯ port echo depends:py36-immutables
py36-contextvars
```

so you'll have to (recursively) remove dependents so that nothing is left in a broken state

Reply:
Right, I only looked into py37- details and assumed that earlier ones do not have dependents either.